src/components/screens/WelcomeScreen.tsx

```diff
@@ -0,0 +1,99 @@
+import { NativeStackNavigationProp } from "@react-navigation/native-stack";
+import FreighterLogo from "assets/logos/freighter-logo-dark.svg";
+import { BaseLayout } from "components/layout/BaseLayout";
+import { Button, ButtonSizes, ButtonVariants } from "components/sds/Button";
+import { Display, Text } from "components/sds/Typography";
+import { AUTH_STACK_ROUTES, AuthStackParamList } from "config/routes";
+import { px } from "helpers/dimensions";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { View } from "react-native";
+import styled from "styled-components/native";
+
+type WelcomeScreenProps = {
+  navigation: NativeStackNavigationProp<
+    AuthStackParamList,
+    typeof AUTH_STACK_ROUTES.WELCOME_SCREEN
+  >;
+};
+
+const Container = styled.View`
+  flex: 1;
+  justify-content: space-between;
+  padding-horizontal: ${px(24)};
+`;
+
+const StyledDisplay = styled(Display)`
+  text-align: center;
+`;
+
+const StyledIconContainer = styled.View`
+  align-items: center;
+  margin-top: ${px(32)};
+`;
+
+const StyledTermsTextContainer = styled.View`
+  margin-bottom: ${px(70)};
+`;
+
+const StyledTermsText = styled(Text)`
+  text-align: center;
+  padding-horizontal: ${px(32)};
+`;
+
+interface StyledProps {
+  $size: number;
+}
+
+const StyledSpacer = styled.View<StyledProps>`
+  height: ${({ $size }: StyledProps) => px($size)};
+`;
+
+export const WelcomeScreen: React.FC<WelcomeScreenProps> = ({ navigation }) => {
+  const { t } = useTranslation();
+
+  const handleCreateNewWallet = () => {
+    navigation.push(AUTH_STACK_ROUTES.CHOOSE_PASSWORD_SCREEN);
+  };
+
+  const handleIAlreadyHaveWallet = () => {
+    navigation.push(AUTH_STACK_ROUTES.IMPORT_WALLET_SCREEN);
+  };
+
+  return (
+    <BaseLayout useSafeArea>
+      <Container>
+        <StyledIconContainer>
+          <FreighterLogo width={px(32)} height={px(32)} />
+        </StyledIconContainer>
+        <View>
+          <StyledDisplay>Freighter Wallet</StyledDisplay>
```

Comment:
Although it doesn't necessarily needs a translation, I think it could be useful to still wrap it with something like `t("freighterWallet")` so if we need to change the app name in the future we could simply change it in the translations files. Wdyt?